lively.collab/comments/commentMorph.js

```diff
@@ -70,31 +69,38 @@ export class CommentGroupMorph extends Morph {
   }
 
   getCommentMorphCount () {
-    return this.ui.commentMorphContainer.submorphs.length;
+    return this.commentMorphs.length;
   }
 
   getUnresolvedCommentMorphCount () {
```

Comment:
I am not sure about our naming scheme here: Technically, we don't resolve Comment Morphs, but comments. Should we name it something like `getUnresolvedCommentCount` or `numberOfUnresolvedComments`?
Same thing above with `getCommentMorphCount`. I am really not sure about it, but wanted to share my thoughts